lib/exqlite/connection.ex

```diff
@@ -460,6 +461,30 @@ defmodule Exqlite.Connection do
     set_pragma(db, "busy_timeout", Pragma.busy_timeout(options))
   end
 
+  defp load_extensions(db, options) do
+    global_extensions = Application.get_env(:exqlite, :load_extensions, [])
+
+    extensions =
+      Keyword.get(options, :load_extensions, [])
+      |> Enum.concat(global_extensions)
+      |> Enum.uniq()
+
+    case extensions do
+      [] ->
```

Comment:
Figured it was worth the check and no-op, to avoid hitting the Sqlite3.enable_load_extension true/false sequence for an empty list.